ansible/roles/common/tasks/users.yml

```diff
@@ -58,6 +58,7 @@
 # remove them.
 - name: remove old group memberships
   command: "gpasswd -d {{ item.member | quote }} {{ item.group | quote }}"
+  when: not keep_old_user_permissions_on_redeploy
```

Comment:
I don't think we should just disable this step, as that would prevent removing sudo users (that are only managed in ansible) and practically result in us forgetting to remove those perms.

A better approach here would be to add (only on the dev-desktop machines) [an extra group of users allowed to ssh](https://github.com/rust-lang/simpleinfra/blob/master/ansible/roles/common/templates/ssh/sshd_config#L37) called something like `dev-desktop-allow-ssh`. That way it won't interfere with the ansible group management.

Reply:
@oli-obk this is still pending btw :slightly_smiling_face: